db/migrate/20150819105725_add_lookup_value_match_to_host_and_hostgroup.rb

```diff
@@ -5,7 +5,7 @@ def up
     Host::Managed.reset_column_information
     Hostgroup.reset_column_information
 
-    Host::Managed.where(:type => 'Host::Managed').find_in_batches(:batch_size => 100) do |group|
+    Host::Managed.where(:type => 'Host::Managed').includes(:primary_interface, :hostgroup, :interfaces, :domain, :operatingsystem).find_in_batches(:batch_size => 1000) do |group|
```

Comment:
fqdn goes to the primary interfaces, which uses data from interfaces, domain, os and hostgroup to retrieve the FQDN. it really shouldn't use os and hostgroup, nor other interfaces, but it does. This PR's scope doesn't require me to figure out what's the deal with these bunch of queries, only to make this migration quicker, which it did, taking the time from roughly 11 hours to 30 minutes.